local_db.py

```diff
@@ -326,8 +322,67 @@
         )
     ]
 
+    # Create some madagascar accounts.
+    users += [
+        User(
+            'madagascar-reports',
+            'reports@madagascartest.org.uk',
+            ('$pbkdf2-sha256$29000$UAqBcA6hVGrtvbd2LkW'
+             'odQ$4nNngNTkEn0d3WzDG31gHKRQ2sVvnJuLudwoynT137Y'),
+            ['madagascar', 'madagascar'],
+            ['reports', 'cd'],
+            data={'name': {'value': 'Report Person'}},
+            state='new'
+        ), User(
+            'madagascar-dashboard',
+            'clinic@madagascartest.org.uk',
+            ('$pbkdf2-sha256$29000$UAqBcA6hVGrtvbd2LkW'
+             'odQ$4nNngNTkEn0d3WzDG31gHKRQ2sVvnJuLudwoynT137Y'),
+            ['madagascar', 'madagascar'],
+            ['dashboard', 'cd'],
+            data={'name': {'value': 'Dashboard Person'}},
+            state='new'
+        ), User(
+            'madagascar-explore',
+            'central.admin@madagascartest.org.uk',
+            ('$pbkdf2-sha256$29000$UAqBcA6hVGrtvbd2LkW'
+             'odQ$4nNngNTkEn0d3WzDG31gHKRQ2sVvnJuLudwoynT137Y'),
+            ['madagascar', 'madagascar'],
+            ['explore', 'cd'],
+            data={'name': {'value': 'Explore Person'}},
+            state='new'
+        ), User(
+            'madagascar-download',
+            'central.admin@madagascartest.org.uk',
+            ('$pbkdf2-sha256$29000$UAqBcA6hVGrtvbd2LkW'
+             'odQ$4nNngNTkEn0d3WzDG31gHKRQ2sVvnJuLudwoynT137Y'),
+            ['madagascar', 'madagascar'],
+            ['download', 'cd'],
+            data={'name': {'value': 'Download Person'}},
+            state='new'
+        ), User(
+            'madagascar-admin-download',
+            'central.admin@madagascartest.org.uk',
+            ('$pbkdf2-sha256$29000$UAqBcA6hVGrtvbd2LkW'
+             'odQ$4nNngNTkEn0d3WzDG31gHKRQ2sVvnJuLudwoynT137Y'),
+            ['madagascar', 'madagascar', 'madagascar'],
+            ['download', 'admin', 'cd'],
+            data={'name': {'value': 'Central Administrator'}},
+            state='new'
+        ), User(
+            'malaria',
+            'malaria@madagascartest.org.uk',
+            ('$pbkdf2-sha256$29000$UAqBcA6hVGrtvbd2LkW'
+             'odQ$4nNngNTkEn0d3WzDG31gHKRQ2sVvnJuLudwoynT137Y'),
```

Comment:
Password hashes in a public repo? Doesn't look like a good idea.

Reply:
This is just for the dev db.  It's fine - the password is just "password" and it doesn't give you access to any live data or information.

Reply:
My bad. I've only looked at file name `local_db.py` and thought it the local dynamodb setup. The domain `madagascartest.org.uk` should also be a good hint for me. Thanks!

Reply:
To be clear- We currently create a "fake" dev db locally for auth (and hermes actually) that allows us to test out and play with the auth functionality without ever infringing on live data - this script local_db.py sets that db up and creates a number of useful accounts roles that all have the same password: "password".